portfolio_ahardman.html

```diff
@@ -0,0 +1,71 @@
+<!doctype html>
+<html lang='en'>
+<head>
+	<title>portfolio_ahardman</title>
+	<link href='normalize.css' rel='stylesheet' type='text/css'>
+	<link href='portfolio_ahardman_stylesheet.css' rel='stylesheet' type='text/css'>
+</head>
+<body>
+	<div class='navbar'>
+		<div class='left container'>
+		</div>
+		<div class='link container'>
+			<a href='#' alt='test_link_1'>Home</a>
```

Comment:
no name given to home after #

Reply:
If you don't assign an id the link goes to the top of the webpage